_includes/footer.html

```diff
@@ -1,250 +1,141 @@
-			allLogos.forEach(e => e.src = e.dataset.logoForDark)
-		} else {
-			layer5Logos.forEach(e => e.src = '../assets/images/company-logo/layer5-no-trim.svg')
-			allLogos.forEach(e => e.src = e.dataset.logoForLight)
-		}
-		if (document.body.classList.contains("dark-mode")) {
-			localStorage.setItem("mode", "dark-mode")
-		} else {
-			localStorage.setItem("mode", "light-mode")
-		}
-		document.dispatchEvent(new CustomEvent("themeChange",{
-			detail : {value : localStorage.getItem("mode")},
-			bubbles: true ,
-		}))
-	}
-	
-</script>
-
-<script src="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.js"></script>
-<script>
-  $(document).ready(function () {
-    $('.slider').slick({
-
-      dots: false,
-      arrows: false,
-      infinite: true,
-      speed: 400,
-      slidesToShow: 3,
-      slidesToScroll: 1,
-      autoplay: true, // true
-      autoplaySpeed: 1500,
-      
-
-      responsive: [
-        {
-          breakpoint: 1300,
-          settings: {
-            slidesToShow: 2.5,
-          }
-        },
-        {
-          breakpoint: 1200,
-          settings: {
-            slidesToShow: 2,
-          }
-        },
-        {
-          breakpoint: 1024,
-          settings: {
-            slidesToShow: 2,
-          }
-        },
-        {
-          breakpoint: 800,
-          settings: {
-            slidesToShow: 1.5,
-            slidesToScroll: 1,
-          }
-        },
-        {
-          breakpoint: 600,
-          settings: {
-            slidesToShow: 1,
-            slidesToScroll: 1,
-            autoplaySpeed: 2000,
-          }
-        },
-        {
-          breakpoint: 480,
-          settings: {
-            slidesToShow: 1,
-            slidesToScroll: 1
-          }
-        },
-        {
-          breakpoint: 400,
-          settings: {
-            slidesToShow: 1,
-            adaptiveWidth: true,
-            slidesToScroll: 1
-          }
-        },
-        {
-          breakpoint: 380,
-          settings: {
-            slidesToShow: 1,
-            slidesToScroll: 1,
-            adaptiveWidth: true,
-          }
-        },
-        {
-          breakpoint: 360,
-          settings: {
-            slidesToShow: 1,
-            slidesToScroll: 1,
-            adaptiveWidth: true,
-          }
-        }
-      ]
-    });
-  });
-</script>
-</body>
-
-</html>
+	if (document.body.classList.contains("dark-mode") ^ (currentMode === "dark-mode")) { setMode(); }
+	function setMode() { /* function logic */ }
+  </script>
+  <script src="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.js"></script>
+  <script>
+	$(document).ready(function () {
+	  $('.slider').slick({
+		dots: false,
+		arrows: false,
+		infinite: true,
+		speed: 400,
+		slidesToShow: 3,
+		slidesToScroll: 1,
+		autoplay: true,
+		autoplaySpeed: 1500,
+		responsive: [{ breakpoint: 1300, settings: { slidesToShow: 2.5 } }, /* other breakpoints */]
+	  });
+	});
+  </script>
```

Comment:
any reasons to remove other breakpoints ? @snehaio 

<img src="https://github.com/user-attachments/assets/fe678cbc-d650-41b8-9317-803ceb63c7ca" width="300px" height="666px" />